owntone.conf.in

```diff
@@ -383,6 +383,16 @@ spotify {
 #	album_override = false
 }
 
+# RCP/Roku Soundbridge output settings
+# Allows OwnTone to output audio to a Roku Soundbridge as a dumb endpoint
```

Comment:
Not sure any users will understand what "dumb endpoint" means. I suggest just deleting this line.